src/box/vy_run.c

```diff
@@ -249,6 +322,7 @@ vy_page_info_create(struct vy_page_info *page_info, uint64_t offset,
 		    const char *min_key, struct key_def *cmp_def)
 {
 	memset(page_info, 0, sizeof(*page_info));
+	rlist_create(&page_info->in_lru);	//temp/ totally unnecessary
```

Comment:
We don't actually have to link the `vy_page_info` node to itself while constructing it, because we never check the LRU list condition via its nodes' members. So the only thing that can be done to a newborn page_info (except destruction) is including it to the list, which doesn't require the node to be self-linked